.gitignore

```diff
@@ -85,5 +85,6 @@ var/lib/node_*
 .idea/
 *.iws
 .DS_Store
+node_modules/*
```

Comment:
what is the purpose of ignoring this folder? (as i don't see nodejs code being added here)

Reply:
I just need to run the `ship_test` to make sure everything works. To run the `ship_test` requires you to do an `npm install`. It would install the javascript modules on the `node_modules` directory under the source tree rather than the build tree.